topofeatures/topofeatures.tex

```diff
@@ -327,13 +327,13 @@ \subsection{Peak}
 that is one point that happens to be a few centimetres higher than all its neighbours would be classified as a peak (the small terrain in Figure~\ref{fig:feature_points} contains several peaks), while if we consider a hill we would surely consider only the top as the peak.
 A peak is therefore on the scale of the data.
 
-The contour line through the $p$ does not exist.
+The contour line through $p$ does not exist.
```

Comment:
@ZokszY the correct way to write this is without the "the" in my opinion. $p$ refers to a specific point.

Reply:
This is what I've done no? I removed the "the"

Reply:
wow, I shouldn't do this while eating my cereals on a Saturday...